waspc/src/Wasp/Generator/ServerGenerator.hs

```diff
@@ -148,7 +149,7 @@ genNvmrc =
     C.mkTmplFdWithDstAndData
       (asTmplFile [relfile|nvmrc|])
       (asServerFile [relfile|.nvmrc|])
-      (Just (object ["nodeVersion" .= ('v' : nodeVersionAsText)]))
+      (Just (object ["nodeVersion" .= show (getMajor nodeVersion)]))
```

Comment:
We only want to specify the major version here. If we specified the entire version string (i.e., `16.0.0`), our project would work only for that _exact_ version, which we probably don't want. Unfortunately, the format doesn't allow me specify a semver compatibility string (i.e., `^16.0.0`, the thing I put inside _package.json_) so this was the next best thing.

I removed the "v" because it was redundant.

Reply:
Hm interesting! This comment is pretty important, and likely something that future reader of this piece of code might benefit from, so I would suggest adding this comment into the code.